drupal/web/themes/custom/pori_events/templates/content/node--event--full.html.twig

```diff
@@ -139,7 +139,7 @@
 
        {% if description %}
         <div class="event__content__description">
-          {{ description| striptags| raw }}
+          {{ description | striptags | raw }}
```

Comment:
Probably works but when piping filters omit the spaces. That is, rather use `{{ description|striptags|raw }}`